.env.example

```diff
@@ -24,3 +24,7 @@
 # EUROPEANA_ENTITIES_API_KEY=apidemo
 # GEONAMES_API_USERNAME=user@example.com
 # SECRET_KEY_BASE=`bundle exec rake secret`
+
+# Recaptcha keys
+# RECAPTCHA_SITE_KEY=
```

Comment:
If this key is not set in the environment, then requesting /migration/new results in failure with the exception:
```
Recaptcha::RecaptchaError at /migration/new
No site key specified.
```

Is this intended? i.e. should setting this key be *required*?

Reply:
since not being a robot is part of the required validation(for non signed in users), the recaptcha keys are required. However, I agree that it should be handled more gracefully when a the value isn't set.